src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/DotNetCommands.cs

```diff
@@ -5,7 +5,7 @@
 using System.IO;
 using System.Runtime.InteropServices;
 
-namespace Microsoft.AspNetCore.Server.IntegrationTesting.Common
+namespace Microsoft.AspNetCore.Server.IntegrationTesting
```

Comment:
Why was this changed?

Reply:
I added this class in my last PR and then realized that it didn't match the namespace for anything else in this folder.